tests/cpp_api/audio_tests/audio_tests.cpp

```diff
@@ -30,11 +30,54 @@ THE SOFTWARE.
 #include <fstream>
 #include <iostream>
 #include <vector>
+#include <cmath>
 
 #include "rocal_api.h"
 
 using namespace std::chrono;
 
+bool verify_non_silent_region_output(int *nsr_begin, int *nsr_length, std::string case_name) {
+    bool pass_status = false;
+    // read data from golden outputs
+    const char *rocal_data_path = std::getenv("ROCAL_DATA_PATH");
```

Comment:
why are we checking for path separately for each node test? this can be done once in common

Reply:
Moved this to test function and removed all other instances